x-pack/legacy/plugins/task_manager/task.ts

```diff
@@ -94,13 +94,25 @@ export interface TaskDefinition {
   description?: string;
 
   /**
-   * How long, in minutes, the system should wait for the task to complete
+   * How long, in minutes or seconds, the system should wait for the task to complete
    * before it is considered to be timed out. (e.g. '5m', the default). If
    * the task takes longer than this, Kibana will send it a kill command and
    * the task will be re-attempted.
    */
   timeout?: string;
 
+  /**
+   * Up to how many times the task should retry when it fails to run. This will
+   * default to the global variable.
+   */
+  maxAttempts?: number;
```

Comment:
> Up to how many times the task should retry when it fails to run

This raises a followup per my last question. maxAttempts is first try + subsequent retries, correct?

Reply:
That's correct, so if you want it to retry two more times when failing, `maxAttempts` would now be set to `3`.